scripts/pr_check.sh

```diff
@@ -0,0 +1,152 @@
+#!/bin/bash
+
+# This script is designed to ensure all relevant header and source files contain correct copyright
+
+set +e
+
+sudo apt-get install dos2unix
+
+ok=0
+fixed=0
+
+function check_folder {
+    for filename in $(find $1 -type f \( -iname \*.cpp -o -iname \*.h -o -iname \*.hpp -o -iname \*.js -o -iname \*.bat -o -iname \*.sh -o -iname \*.txt \)); do
+          if [[ $(grep -oP "Software License Agreement" $filename | wc -l) -ne 0 ]]; then
+               echo "[WARNING] $filename contains 3rd-party license agreement"
+          else
+               if [[ ! $filename == *"libuvc"* ]]; then
+                    # Only check files that are not .gitignore-d
+                    if [[ $(git check-ignore $filename | wc -l) -eq 0 ]]; then
+                         if [[ $(grep -oP "(?<=\(c\) )(.*)(?= Intel)" $filename | wc -l) -eq 0 ]]; then
+                              echo "[ERROR] $filename is missing the copyright notice"
+                              ok=$((ok+1))
```

Comment:
@dorodnic  why not $ok = 1?

Reply:
I want to count the number of errors to print it in the end.
This helps the user, as he can see this number get lower as he works through the issues.